lib/xterm.sh

```diff
@@ -0,0 +1,32 @@
+# script taken straight from bash-it with almost no adaptation
```

Comment:
First of all, this xterm.sh and bash-preexec.sh seem to be actually completely independent of the suggested theme ab+simple. Also, the current setup forces to set up the xterm titles regardless of the current theme. I have checked the original Bash-it and found that it is actually a plugin in Bash-it but not a basic component of the framework loaded by default. I'll just move the file to `plugins/xterm.plugin.sh`.

Reply:
done